src/active-response/firewalls/pf.c

```diff
@@ -101,6 +110,35 @@ int main (int argc, char **argv) {
                 snprintf(log_msg, OS_MAXSTR - 1, "Table '%s' does not exist", PFCTL_TABLE);
                 write_debug_file(argv[0], log_msg);
                 cJSON_Delete(input_json);
+
+                (void)write_cmd_to_file(RCCONF, "pf_enable=\"YES\"" "\n" "pf_rules=\"/etc/pf.conf\"" "\n" "pflog_enable=\"YES\"" "\n" "pflog_logfile=\"/var/log/pflog\"");
```

Comment:
Why don't you verify the return value?

Reply:
`"pf_enable=\"YES\"\npf_rules=\"/etc/pf.conf\"\npflog_enable=\"YES\"\npflog_logfile=\"/var/log/pflog\""`